src/redux/reducers/bookReducer.test.js

```diff
@@ -0,0 +1,30 @@
+import bookReducer from './bookReducer';
+import { FETCH_BOOKS } from '../actions/types';
+
+describe('Books reducer', () => {
+  it('should return the initial state', () => {
+    expect(bookReducer(undefined, {})).toEqual({
+      books: [],
+    });
+  });
+
+  it('should handle FETCH_BOOKS', () => {
```

Comment:
Would it be good to have a case where state has some other properties to check that it doesn't interfere with them and only manages books ?